### PR TITLE
Actually go vet.

### PR DESCRIPTION
**Pull request by spxtr**
#3756

### CODE REVIEW

**Comment by ixdy** [comment]
`gcloud-in-go:0.1` is based off `golang:1.7.4`. might need to update that...

File: verify/verify-govet.sh [hunk 1]
```diff
@@ -17,4 +17,6 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-find . -name "*.go" | grep -v "\/vendor\/" | xargs -n 1 go vet
+go get ./...
```
Comment (ixdy) [inline verify/verify-govet.sh]: do we need this? or just the `go install` line?

Comment (spxtr) [inline verify/verify-govet.sh]: The aws-janitor doesn't vendor its dependencies.

Comment (spxtr) [inline verify/verify-govet.sh]: (And I have to stop shaving yaks at some point, my whole goal is to fix races in mungegithub but I need go vet to be working to do that)

**Comment by ixdy** [comment]
looks like it's now failing as expected?
/lgtm

**Comment by spxtr** [comment]
I'm going to push the fixes as a second commit to this PR.

**Comment by spxtr** [comment]
Passed :) PTAL @ixdy

**Pull request merged**